pkg/costattribution/active_tracker.go

```diff
@@ -34,6 +34,14 @@ type ActiveSeriesTracker struct {
 	overflowCounter atomic.Int64
 }
 
+func addLabelsPrefix(labels []string) []string {
+	out := make([]string, 0, len(labels))
+	for _, l := range labels {
+		out = append(out, strings.Join([]string{usagePrefix, l}, ""))
```

Comment:
This should be just:
```suggestion
		out = usagePrefix + l
```